src/components/DashBoard.js

```diff
@@ -29,6 +32,29 @@ class DashBoard extends Component {
 
   render() {
     const { classes, showFullMap } = this.props;
+    const chartContent = {
+      title: 'Live Coral Cover',
+      body: [
```

Comment:
This stuff... It's soon going to be replaced by an API, correct?

Reply:
yes, absolutely. I just create some dummy data to play around with the victory bar chart.

Reply:
Awesome. Make a `constants/`-folder, and jam that data into a `sampleChartData` file please :)